bot/constants.py

```diff
@@ -123,11 +119,11 @@ def check_required_keys(keys):
                 if lookup is None:
                     raise KeyError(key)
         except KeyError:
-            log.critical(
+            raise (
```

Comment:
Need an exception class here, `KeyError` would probably be good, So `raise KeyError("message")`. That or we could print and then just `raise`, maybe that would be better as it would keep the context of where the KeyError was raised? Can't remember exactly how these things work